lean.bib

```diff
@@ -33,24 +33,24 @@ @InProceedings{	  AvigadCarneiroHudon19
 }
 
 @InProceedings{	  Baanen20,
-  author    = {Anne Baanen},
-  editor    = {Nicolas Peltier and
-               Viorica Sofronie{-}Stokkermans},
-  title     = {A Lean Tactic for Normalising Ring Expressions with Exponents (Short
-               Paper)},
-  booktitle = {Automated Reasoning - 10th International Joint Conference, {IJCAR}
-               2020, Paris, France, July 1-4, 2020, Proceedings, Part {II}},
-  series    = {Lecture Notes in Computer Science},
-  volume    = {12167},
-  pages     = {21--27},
-  publisher = {Springer},
-  year      = {2020},
-  url       = {https://doi.org/10.1007/978-3-030-51054-1\_2},
-  doi       = {10.1007/978-3-030-51054-1\_2},
-  timestamp = {Thu, 06 Aug 2020 21:49:45 +0200},
-  biburl    = {https://dblp.org/rec/conf/cade/Baanen20.bib},
-  bibsource = {dblp computer science bibliography, https://dblp.org},
-  tags      = {about-mathlib, lean3}
+  author	= {Anne Baanen},
```

Comment:
Any reason to modify the formatting of this entry and `SelsamUllrichDeMoura20` ? LGTM otherwise.

Reply:
I ran the formatting incantation that Patrick told me a while back, and this is what it did...

Reply:
Hmm, it looks ragged in the diff view, but when I [view file](https://github.com/leanprover-community/leanprover-community.github.io/blob/12139a7a8b8eeb32b6e5e759807cb16a115bc3b1/lean.bib) it looks fine.